Coordinator/Coordinator/Coordinator/AppCoordinator.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  AppCoordinator.swift
+//  Coordinator
+//
+//  Created by Vladka on 28.12.2022.
+//
+
+import Foundation
+
+/// Батя координатор
```

Comment:
```suggestion
/// Батя координатор
```
не уверена что это можно оставить)))😂

Reply:
в Новый год можно все)